### PR TITLE
New functions + some updates to the old ones

### PR DESCRIPTION
**Pull request by DmitrySharabin**

### CODE REVIEW

**Comment by DmitrySharabin** [comment]
Here is the first pass on updating the list of supported functions.

@LeaVerou Lea, will you please review it when you have a couple of minutes of free time?

## Updated Functions

- [`date()`](https://deploy-preview-146--mavoio.netlify.app/docs/functions/?functions-storage=https://deploy-preview-146--mavoio.netlify.app/docs/functions/functions.json#date)
- [`time()`](https://deploy-preview-146--mavoio.netlify.app/docs/functions/?functions-storage=https://deploy-preview-146--mavoio.netlify.app/docs/functions/functions.json#time)

## New Functions
- [`datetime()`](https://deploy-preview-146--mavoio.netlify.app/docs/functions/?functions-storage=https://deploy-preview-146--mavoio.netlify.app/docs/functions/functions.json#datetime)
- [`readable_datetime()`](https://deploy-preview-146--mavoio.netlify.app/docs/functions/?functions-storage=https://deploy-preview-146--mavoio.netlify.app/docs/functions/functions.json#readable_datetime)
- [`intersection()`](https://deploy-preview-146--mavoio.netlify.app/docs/functions/?functions-storage=https://deploy-preview-146--mavoio.netlify.app/docs/functions/functions.json#intersection)
- [`pluralize()`](https://deploy-preview-146--mavoio.netlify.app/docs/functions/?functions-storage=https://deploy-preview-146--mavoio.netlify.app/docs/functions/functions.json#pluralize)
- [`sort()`](https://deploy-preview-146--mavoio.netlify.app/docs/functions/?functions-storage=https://deploy-preview-146--mavoio.netlify.app/docs/functions/functions.json#pluralize)

File: docs/functions/functions.json [hunk 1]
```diff
@@ -1452,6 +1585,77 @@
...
+					"name": "n",
+					"optional": false,
+					"type": "Number",
+					"description": "Number to get the plural form of the word for."
+				},
+				{
+					"name": "rule1",
+					"optional": false,
+					"type": "Text | Key:Value pair",
+					"description": "Pluralization rule to apply. If there is only one plural form of a word in the language, this parameter should provide the **singular form** of the word."
```
Comment (LeaVerou) [inline docs/functions/functions.json:1606]:
```suggestion
					"description": "Pluralization rule to apply. If there is only one plural form of a word in the language (which is the case in English), this parameter should provide the **singular form** of the word."
```

File: docs/functions/functions.json [hunk 2]
```diff
@@ -1452,6 +1585,77 @@
...
+					"name": "rule1",
+					"optional": false,
+					"type": "Text | Key:Value pair",
+					"description": "Pluralization rule to apply. If there is only one plural form of a word in the language (which is the case in English), this parameter should provide the **singular form** of the word."
+				},
+				{
+					"name": "rule2",
+					"optional": false,
+					"type": "Text | Key:Value pair",
+					"description": "Pluralization rule to apply. If there is only one plural form of a word in the language, this parameter should provide the **plural form** of the word."
```
Comment (LeaVerou) [inline docs/functions/functions.json:1612]:
```suggestion
					"description": "Pluralization rule to apply. If there is only one plural word form in the language (which is the case in English), this parameter should provide the **plural form** of the word."
```

**Comment by LeaVerou** [comment]
Fantastic work, thanks Dmitry!!

**Pull request merged**